jenkins-scripts/dsl/ignition.dsl

```diff
@@ -63,8 +63,7 @@ ignition_debbuild = ignition_software + [ 'cmake3',
                                           'rendering5',
                                           'sensors5',
                                           'tools2',
-                                          'transport10',
-                                          'utils1']
+                                          'transport10']
```

Comment:
This is needed for the nightlies, no?

Reply:
I believe the nightlies are using https://build.osrfoundation.org/job/ign-utils-debbuilder/

the removed line created this job https://build.osrfoundation.org/job/ign-utils1-debbuilder/ , which was used at one point but looks like it's not part of the nightlies

Reply:
I see, I guess line 45 is wrong, because utils isn't stable yet, and here we should have `utils` instead of `utils1`?

Reply:
yeah, it's a little premature to have it on line 45; the following job is failing

* https://build.osrfoundation.org/job/ignition_utils-ci-ign-utils1-bionic-amd64/

but I'm not sure if there's any other way to get the `main` branch jobs created

yeah, maybe this line should be `utils`

Reply:
How about 3b0ae22ce701c5574008c85d249891707b6b3f81? Based on comment https://github.com/ignition-tooling/release-tools/pull/389/files#diff-1269f38223f9cd477137c0a99b7e9ff46f8eeb3b09300d2b245ec8f9381172f5R29 .